### PR TITLE
Added Topiclist

### PR DESCRIPTION
**Pull request by Miya25**
Added Topiclist

### CODE REVIEW

File: data/lists/topiclist.xyz.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+    "id": "topiclist.xyz",
+    "name": "TopicList",
+    "url": "https://topiclist.xyz/",
+    "icon": "https://cdn.topiclist.xyz/images/png/TopicList5.png",
+    "language": "English",
+    "display": 1,
+    "defunct": 0,
+    "discord_only": 0,
```
Comment (Andre601) [inline data/lists/topiclist.xyz.json:9]: Is the list only listing discord bots? Then this should be 1

File: data/lists/topiclist.xyz.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "defunct": 0,
+    "discord_only": 0,
+    "description": "Do you want to expand and improve your Discord bot? TopicList is here for you!",
+    "api_docs": "https://docs.topiclist.xyz",
+    "api_post": "https://api.topiclist.xyz/bot/:id/stats",
+    "api_field": "guilds",
+    "api_shard_id": null,
+    "api_shard_count": "shards",
+    "api_shards": null,
+    "api_get": "https://topiclist.xyz/api/bots/:botID",
```
Comment (MattIPv4) [inline data/lists/topiclist.xyz.json:17]:
```suggestion
    "api_get": "https://topiclist.xyz/api/bots/:id",
```
The placeholder for the bot ID should always be `:id`

**Review by MattIPv4 — CHANGES_REQUESTED** [review]
You have multiple CI failures that need addressing:

- You are missing the `$schema` property
- You are missing the `added` property
- You have included invalid items in `features`
- You have line-endings that do not match the editorconfig for the repo

**Comment by Miya25** [comment]
hmm it should work now..

File: data/lists/topiclist.xyz.json (hunk 1 above)
Comment (Miya25) [reply data/lists/topiclist.xyz.json:9]: but it also have serverlisting..

Comment (Andre601) [reply data/lists/topiclist.xyz.json:9]: The option is not about if you only allow discord bots, but if you only allow discord-related content.
If your site is also listing servers and bots from other services (i.e. revolt.chat) would this being 0 be okay. But if your site only allows discord-related stuff then it has to be 1. It's that simple.

**Comment by Miya25** [comment]
@MattIPv4 any updates on this?

**Comment by MattIPv4** [comment]
@Miya25 You still have CI failures, the same CI failures from over a month ago. If you do not solve them, I am going to have to close this PR due to a lack of progress.

**Comment by Miya25** [comment]
hmm i am trying to some it but idk whats causing it

**Comment by MattIPv4** [comment]
As the CI errors note:

1. Your file does not match the editorconfig for this repo, check that your line endings are correct.

2. You are missing the `added` property from your file, which is required per the schema.

3. You have listed features for your list that are not recognised features on BotBlock, based on [data/features](https://github.com/botblock/data/tree/master/data/features).

4. You are missing the `$schema` property from your file.

**Comment by MattIPv4** [comment]
Closing, no progress.

**Pull request closed without merging**